# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1421647452 -3600
#      Mon Jan 19 07:04:12 2015 +0100
# Branch gui-release
# Node ID 6d75f1683ce822a2fae79be0d3086bb801ec1bcf
# Parent  4fe86a372f10c3143a326bcae0f5c4bac63a4dd7
fix scroll width of horizontal scroll bar (bug #44040)

*file-editor-tab.cc (notice-settings): enable scroll width tracking

diff --git a/libgui/src/m-editor/file-editor-tab.cc b/libgui/src/m-editor/file-editor-tab.cc
--- a/libgui/src/m-editor/file-editor-tab.cc
+++ b/libgui/src/m-editor/file-editor-tab.cc
@@ -1903,16 +1903,17 @@ file_editor_tab::notice_settings (const 
         (settings->value ("editor/indent_width",2).toInt ());
 
   _edit_area->setTabWidth
         (settings->value ("editor/tab_width",2).toInt ());
 
   _edit_area->SendScintilla (QsciScintillaBase::SCI_SETHSCROLLBAR,
         settings->value ("editor/show_hscroll_bar",true).toBool ());
   _edit_area->SendScintilla (QsciScintillaBase::SCI_SETSCROLLWIDTH,-1);
+  _edit_area->SendScintilla (QsciScintillaBase::SCI_SETSCROLLWIDTHTRACKING,true);
 
   _long_title = settings->value ("editor/longWindowTitle", false).toBool ();
   update_window_title (_edit_area->isModified ());
 
   _edit_area->setEdgeColumn (
               settings->value ("editor/long_line_column",80).toInt ());
   if (settings->value ("editor/long_line_marker",true).toBool ())
     _edit_area->setEdgeMode (QsciScintilla::EdgeLine);
